gnatsd/ci/gnatsd.rake

```diff
@@ -0,0 +1,90 @@
+require 'ci/common'
+
+def gnatsd_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def gnatsd_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/gnatsd_#{gnatsd_version}"
+end
+
+def gnatsd_temp_dir
+  ENV['VOLATILE_DIR'] + '/gnatsd'
+end
+
+container_name = 'dd-test-gnatsd'
+
+namespace :ci do
+  namespace :gnatsd do |flavor|
+    task before_install: ['ci:common:before_install'] do
+      Rake::Task['ci:gnatsd:cleanup'].invoke
+    end
+
+    task :install do
+      Rake::Task['ci:common:install'].invoke('gnatsd')
+      volumes = "--volume=#{gnatsd_temp_dir}/data:/usr/share/nats/data"
+      publish = '--publish=4222:4222 --publish=8222:8222'
+      cluster = '-cluster=nats://0.0.0.0:5222'
+
+      # Set up NATS cluster
+      sh %(docker run -d --name #{container_name}-serverA --publish=5222:5222 nats:1.0.4 #{cluster})
```

Comment:
would not affect the test but `gnatsd` docker image `1.1.0` is already available if want to update to test against latest